src/axom/spin/docs/sphinx/conf.py

```diff
@@ -0,0 +1,282 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file is no longer needed, since all components use the top-level ``conf.py`` file.

Reply:
thanks; deleted